BeforeBuild.cmd

```diff
@@ -10,7 +10,14 @@ SETLOCAL
 dotnet restore src\Everything.sln --packages src\packages
 
 if "%ERRORLEVEL%" NEQ "0" (
-echo nuget restore failed
+echo nuget restore failed.
+goto ExitFailed
+)
+
+:: Generate the SARIF object model classes from the SARIF JSON schema.
+msbuild /verbosity:minimal /target:BuildAndInjectObjectModel src\Sarif\Sarif.csproj /fileloggerparameters:Verbosity=detailed
```

Comment:
Note: we do this in `BeforeBuild.cmd` rather than in `BuildAndTest.cmd` to ensure that this happens in AppVeyor as well as on the desktop.